packages/api-clients/open-api/m2mGatewayApi.yml

```diff
@@ -4281,6 +4401,66 @@ components:
           format: uuid
       required:
         - purposeId
+    ProducerDelegation:
```

Comment:
This model and the one below, `ProducerDelegations`, are identical to the equivalent for consumer delegations (`ConsumerDelegation` and `ConsumerDelegations`). Should we keep them separate because they are different resources from the user's point of view? Or should we have a `Delegation` / `Delegations` model for both?